### DISCUSSION TITLE
Query regarding MICOM's model database

### DISCUSSION THREAD

**Discussion by shreyanshumale** [micom-dev/micom#48] [Q&A] [answered]
Hello !

I have GSMMs from a hydrothermal vent; when I try to run MICOM workflow on these, I don't find enough representation for my models in the MICOM model database. I tried using carveme_refseq databases for all taxonomic levels, but the found_fraction is very low (~0,18). I thought this was because the database would be gut microbiome relevant. Is there a way I could troubleshoot this?

Please let me know!

Thanks,
Shreyansh

**Comment by cdiener** [comment, marked as answer]
Hi, the CARVEME refseq database covers all full and verified genomes in the NCBI database. So if you get low matches this could mean that there are no reference genomes for the organisms you are studying. 
Another thing you could check is whether your classification for the taxa names used a database that uses the NCBI taxonomy. Otherwise, you might get fewer matches if the databases don't match. How did you assign taxonomy to the sequences? If you used SILVA you could retry with an NCBI database. If those were metagenomic data and you used kraken that won't be an issue though...
